markdownlint/docs/TOP010.md

```diff
@@ -0,0 +1,29 @@
+# `TOP010` - Use lazy numbering within ordered lists
+
+Tags: `ordered-list, lazy-numbering`
+
+Aliases: `use-lazy-numbering-in-ordered-lists`
```

Comment:
Should use the name of the rule, which should match the 2nd string in the rule object's `name` array. i.e. `["TOP010", "insert-kebab-case-name-here"]`
Whichever name you feel is best, if you can change things so all reference that same name, that'd be great.